hw_1/vipcls data.R

```diff
@@ -57,10 +59,15 @@ vipcls$bw[vipcls$bw<300] <- NA
 #Set birth weight of babies (bw) 6000 and above to missing 
 vipcls$bw[vipcls$bw>=6000] <- NA
 
-#Convert binary variables to logicals. 
+#Convert binary variables to logicals
+
 #Step 1: Convert deltype cesarean (2) to 0 (false) and convert to logical
-vipcls$deltype[vipcls$deltype==2] <- 0
-vipcls$deltype <- as.logical(vipcls$deltype)
+# Are we considering deltype as boolean? If yes, this code works correctly.
+# Here I've assigned the value to a new variable instead of overwriting the
+# original. 
+vipcls$deltype_bool <- vipcls$deltype
+vipcls$deltype_bool[vipcls$deltype_bool==2] <- 0
+vipcls$deltype_bool <- as.logical(vipcls$deltype_bool)
```

Comment:
Instead of overwriting the original field, I created a new field called `deltype_bool` here that changes the original value into a boolean.